examples/api/test/widgets/color_filter/color_filtered.0_test.dart

```diff
@@ -0,0 +1,67 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_api_samples/widgets/color_filter/color_filtered.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  // The app being tested loads images via HTTP which the test
+  // framework defeats by default.
+  setUpAll(() {
+    HttpOverrides.global = null;
+  });
+
+  testWidgets('Displays two images', (WidgetTester tester) async {
```

Comment:
These 3 tests can be just one.

"'Displays two images" test - you're already finding first and last images the implies are only two images.

"'Applies red color filter with modulate blend mode to the first image"  & "Applies grey color filter with saturation blend mode to the second image" - These two short and too verbose. We can add single test with "Color filter is applied to the images" and explain the difference in the test comment instead.

Reply:
@TahaTesser thanks for the review and great suggestion. I applied it in the latest commit.